src/style/contentstyle.rs

```diff
@@ -0,0 +1,79 @@
+//! This module contains the `content style` that can be applied to an `styled content`.
+
+use std::fmt::Display;
+
+use super::{Attribute, Color, StyledContent};
+
+/// A content style.
+#[derive(Debug, Clone, Default)]
+pub struct ContentStyle {
+    /// The foreground color.
+    pub fg_color: Option<Color>,
+    /// The background color.
+    pub bg_color: Option<Color>,
+    /// List of attributes.
+    pub attrs: Vec<Attribute>,
+}
+
+impl ContentStyle {
+    /// Creates a `StyledContent` by applying the style to the given `val`.
+    pub fn apply<D: Display + Clone>(&self, val: D) -> StyledContent<D> {
+        StyledContent::new(self.clone(), val)
+    }
+
+    /// Creates a new `ContentStyle`.
+    pub fn new() -> ContentStyle {
+        ContentStyle::default()
+    }
+
+    /// Sets the background color.
+    pub fn background(mut self, color: Color) -> ContentStyle {
```

Comment:
Since this is breaking change, `background_color` is way better, but too long. Not sure here, just thinking aloud. Same for the `foreground`.

Reply:
IMHO `background` works well because it already takes a color as parameter and it can't be another thing than the color (at least currently).

Reply:
Okay, let be it.